dbt_project.yml

```diff
@@ -9,7 +9,7 @@ models:
     +materialized: table
     intermediate:
       +materialized: ephemeral
-    staging:
+    staging: 
```

Comment:
What are your thoughts on making the staging models ephemeral? I was initially thinking this would be a good idea, but then I saw the size of our internal data and imagine this may not be ideal.